src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -77,14 +102,14 @@ class RatedDisabilityList extends React.Component {
     if (!this.props.ratedDisabilities) {
       return <LoadingIndicator message="Loading your information..." />;
     }
-    // Display error message based on error type.
     if (
       this.props.ratedDisabilities.errors ||
```

Comment:
should this be `this.props?.ratedDisabilities?.errors` to match the line below it?